CMakeLists.txt

```diff
@@ -380,7 +380,11 @@ endif()
 ################################################################################
 # FIXME: Support ARM "-mfpu=vfp -mfloat-abi=hard"
 if (("${TARGET_ARCHITECTURE}" STREQUAL "x86_64") OR ("${TARGET_ARCHITECTURE}" STREQUAL "i686"))
-  if (("${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU") OR ("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang"))
+  if (("${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU") OR ("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang") OR ("${CMAKE_CXX_COMPILER_ID}" MATCHES "Intel"))
```

Comment:
This would be more clear as a separate block:

```cmake
if("${CMAKE_CXX_COMPILER_ID}" MATCHES "Intel")
  list(APPEND Z3_DEPENDENT_LIBS "iomp5")
  set(SSE_FLAGS "-mfpmath=ss3" "-msse" "-msse2")
elseif(("${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU") OR ("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang"))
  set(SSE_FLAGS "-mfpmath=ss3" "-msse" "-msse2")
elseif("x${CMAKE_CXX_COMPILER_ID"} STREQUAL "xMSVC")
...
```

Reply:
agree